.travis.yml

```diff
@@ -16,11 +30,14 @@ php:
     - 7.0
     - hhvm
 
-before_script:
+before_install:
     - composer self-update
     # disable xdebug since we don't use code-coverage for now
     - if [[ $TRAVIS_PHP_VERSION != 'hhvm' && $TRAVIS_PHP_VERSION != '7.0' ]]; then phpenv config-rm xdebug.ini; fi
+    - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
 
-install: composer install --prefer-source
+install:
+    - composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
```

Comment:
No problem.
And about Github configuration :arrow_right: http://blog.wyrihaximus.net/2015/09/github-auth-token-on-travis/ :wink: